src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -109,15 +109,7 @@ class Favourites extends Component {
 
     toggleFavourite(stockCode) {
```

Comment:
I know this is existing code, but I find it quite confusing that `Favourites` has `toggleFavourite()` and `Search` has `onIconClick()` which do (essentially) the same thing - perhaps they can be renamed to be consistent? `onIconClick` seems the most consistent to me.